content/blog/2025-05-02-upgrade-stories-vol1.md

```diff
@@ -0,0 +1,32 @@
++++
+title= "Upgrade Stories from the Community, Volume 1:" 
+description = "Two new Valkey users describe what it's really like to upgrade" 
+date= 2025-05-02 01:01:01 
+authors= ["kyledvs", "nigel"] 
++++
+
+## Two new Valkey users describe what it's really like to upgrade
+
+
+Many potential Valkey users have told the project that they're interested in hearing more stories about companies that have decided to migrate to Valkey. This blog is the first in a series that will share our users' stories. Two organizations, Muse and 4th Whale Marketing, share a similar story: they both have Valkey in a critical place in their applications, they switched engines quickly with zero issues, and they saved money.
+
+Valkey is a multi-vendor/vendor neutral project. With the support of the a large and diverse community, the project is innovating at a rapid pace. In September 2024, we released Valkey 8.0 which brought performance improvements. Six months later in March 2025, Valkey 8.1 added memory efficiency improvements, extended observability, and more performance improvements. Further, it has a permissive, open source license allowing for anyone to use, modify, and provide Valkey as a Service. This allows organizations to make changes that best fit their values, budgets, and technical constraints. Valkey provides choice and prevents lock-in by being offered across many vendors, Linux distributions, with no licensing costs. 
+
+According to [Muse](https://www.museml.com/), they are,  “the first intelligent creative assistant and inspiration studio for storytellers and world builders.” They use Valkey in several ways:
+
+* Caching: “We utilize standard HTML and payload caching to offload redundant data retrieval and computation, improving performance and efficiency.”
```

Comment:
Who said these? Garth?

Reply:
Correct

Reply:
The top quote is from their website. The bottom ones are from Garth.

Reply:
maybe we can make this a little more clear?

Reply:
Do you have a suggestion? Do we want to add some copy around the quote to clarify it?

Reply:
Wait. I think I understand the ask now.